Makefile

```diff
@@ -29,7 +29,7 @@ PULL_POLICY ?= Always
 MANAGER_IMAGE_NAME ?= cluster-api-azure-controller
 
 # A release should define this with the next version after 0.1.0
-MANAGER_IMAGE_TAG ?= dev
+MANAGER_IMAGE_TAG ?= 0.1.1
```

Comment:
This variable refers to the version of the capz controller manager, not the [Cluster API](https://github.com/kubernetes-sigs/cluster-api/releases/tag/0.1.1) controller manager.
Revert this change when you have a chance.

Reply:
My bad -- didn't mean to commit this.